packages/frontend/editor-ui/src/features/shared/banners/components/banners/DynamicBanner.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import { N8nMarkdown } from '@n8n/design-system';
+import BaseBanner from './BaseBanner.vue';
+
+interface Props {
+	name: string;
+	content: string;
+	theme: 'info' | 'warning' | 'danger';
+	isDismissible: boolean;
+	dismissPermanently: boolean | null;
+}
+
+const props = defineProps<Props>();
+</script>
+
+<template>
+	<BaseBanner
+		:name="props.name"
+		:theme="props.theme"
+		:dismissible="props.isDismissible"
+		:dismiss-permanently="props.dismissPermanently ?? false"
+	>
+		<template #mainContent>
+			<N8nMarkdown :content="props.content" :class="$style.markdown" />
```

Comment:
Comparing this to static banners:
1. Looks like icon is not aligned to markdown content properly:
2. Text is rendered bigger than in existing banners

Dynamic banner:
<img width="421" height="75" alt="image" src="https://github.com/user-attachments/assets/fec15cce-887a-43c0-862f-46daa3b87563" />

Static banner:
<img width="497" height="72" alt="image" src="https://github.com/user-attachments/assets/e71ae822-1045-46a9-b171-92f6af6aadea" />